lib/tasks/cleanup.rake

```diff
@@ -0,0 +1,36 @@
+namespace :foreman_docker do
+  desc 'Clean default data created by this plugin, this will permanently delete the data!'
+  task :cleanup => :environment do
```

Comment:
Perhaps we should make explicit this is destructive. Should we name the task `destroy`?

Reply:
other plugins (templates and virt_who_configure) alread uses cleanup, they don't drop tables but still, consistency++